src/classes/User.js

```diff
@@ -11,8 +11,9 @@ class User {
   }
 
   addFavorite(recipe) {
-    const favRecipe = this.favoriteRecipes;
-    favRecipe.push(recipe);
+    if (!this.favoriteRecipes.includes(recipe)) {
```

Comment:
This function should _only_ add the recipe, not check if the recipe exists or not. Could we move this check somewhere else so we don't _ever_ call `addFavorite` if the favorite is already selected?

Basically, the idea is that, if we shouldn't add the favorite, we should never call `addFavorite`.